aidants_connect_web/templates/public_website/politique_confidentialite.html

```diff
@@ -0,0 +1,189 @@
+      <h2 class="fr-mt-4w">Quelles sont les données que nous traitons&nbsp;?
+      </h2>
+      <p class="fr-mb-1w">Aidants Connect traite les données à caractère personnel suivantes&nbsp;:</p>
+      <ul class="fr-pl-2w">
+        <li><strong>Données d'identification de l'usager&nbsp;:</strong> nom, prénom, date et lieu de naissance, genre, adresse e-mail, alias technique FranceConnect, alias technique Aidants Connect&nbsp;;</li>
+        <li><strong>Données d'identification des aidants habilités par les structures&nbsp;:</strong> nom, prénom, adresse e-mail professionnelle, fonction&nbsp;;</li>
+        <li><strong>Données relatives au mandat&nbsp;:</strong> nom et prénom de l'usager, nom et prénom de l'aidant.</li>
+      </ul>
+      <h2 class="fr-mt-4w">Qu'est-ce qui nous autorise à traiter ces données&nbsp;?</h2>
+      <p class="fr-mb-1w">Aidants Connect traite des données à caractère personnel en se basant sur&nbsp;:</p>
+      <ul class="fr-pl-2w">
+        <li>L'exécution d'une mission d'intérêt public ou relevant de l'exercice de l'autorité publique dont est investi le responsable de traitement au sens de l'article 6-1 e) du RPGD.</li>
+      </ul>
+      <h2 class="fr-mt-4w">Pendant combien de temps conservons-nous ces données&nbsp;?</h2>
+      <div class="fr-table fr-table--bordered fr-table--layout-fixed">
+        <table>
+          <thead>
+            <tr>
+              <th scope="col">Type de données</th>
+              <th scope="col">Durées de conservation</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><strong>Données d'identification de l'usager</strong></td>
+              <td>3 ans en base active après l'expiration de leur dernier mandat et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+            <tr>
+              <td><strong>Données d'identification des aidants habilités par les structures</strong></td>
+              <td>3 ans en base active à compter du dernier contact avec l'aidant et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+            <tr>
+              <td><strong>Données relatives au mandat</strong></td>
+              <td>3 ans en base active à compter de l'expiration du mandat et 3 ans supplémentaire dans l'archivage intermédiaire</td>
+            </tr> 
+          </tbody>
+        </table>
+      </div>
+      <h2 class="fr-mt-4w">Quels droits avez-vous&nbsp;?</h2>
+      <p class="fr-mb-1w">Vous disposez&nbsp;:</p>
+      <ul class="fr-pl-2w fr-mb-1w">
+        <li>D'un droit d'information et d'un droit d'accès à vos données&nbsp;;</li>
+        <li>D'un droit de rectification&nbsp;;</li>
+        <li>D'un droit d'opposition&nbsp;;</li>
+        <li>D'un droit à la limitation du traitement.</li>
+      </ul>
+      <p>Pour les exercer, contactez-nous à&nbsp;: <a href="{% mailto_href AC_CONTACT_EMAIL %}" class="fr-link" target="_blank">contact@aidantsconnect.beta.gouv.fr</a></p>
```

Comment:
```suggestion
      <p>Pour les exercer, contactez-nous à&nbsp;: {% mailto AC_CONTACT_EMAIL %}</p>
```

Pardon, je t'ai donné le mauvais code lors de la dernière revue. `{% mailto %}` génère automatiquement le `<a>`.

Reply:
si je ne mets que `{% mailto %}` ça me déclenche une erreur 
 ```
 TemplateSyntaxError at /cgu/

'mailto' did not receive value(s) for the argument(s): 'recipient'
 ```